content/release-notes/1.61.0.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 2022-01-28T10:21:28-07:00
+linktitle: "1.61.0"
+title: "1.61.0"
+kubernetes: "1.20, 1.21, and 1.22"
+---
+
+{{<features>}}
+* Adds a CLI command to collect all versions for an application. See [the docs](https://kots.io/kots-cli/get/versions/) for more details.
```

Comment:
Do we usually abbreviate as 'the docs'?
Suggest the following:
Adds a CLI command to collect all available versions for an application from app manager. See [the documentation](https://kots.io/kots-cli/get/versions/) for more details.

Reply:
Even shorter, we could hyperlink the CLI command and get rid of the second sentence altogether. (I went with KOTS since we haven't really rebranded the docs to use app manager yet.) And "get" sounds more natural than "collect", especially since the command name looks like `get` (inferred from the docs link).
```suggestion
* Adds a CLI command to [get all available versions](https://kots.io/kots-cli/get/versions/) for an application from KOTS.
```

Reply:
Definitely good recommendations; lgtm